tools/bootnode/bootnode.go

```diff
@@ -155,9 +156,13 @@ func startKademliaDHT(privKey crypto.PrivKey) {
 }
 
 func createListener(ipAddr string, port int, cfg discover.Config) *discover.UDPv5 {
+	ipAddr, err := iputils.ExternalIPv4()
```

Comment:
we are overwriting the input value of the func. maybe it doesnt need an input ipAddr?

Reply:
fixed it